src/fluree/server/consensus/events.clj

```diff
@@ -64,7 +81,9 @@
              :ledger-id ledger-id
              :opts      opts
              :instant   (System/currentTimeMillis)}]
-    (with-txn evt txn)))
+    (if (= :turtle (:format opts))
+      (with-turtle-txn evt txn)
+      (with-txn evt txn))))
```

Comment:
this is minor, but perhaps this function could have a more specific name to further disambiguate it from turtle.